src/app/legacy/containers/CpsRecommendations/__snapshots__/index.test.jsx.snap

```diff
@@ -155,17 +155,6 @@ exports[`CpsRecommendations should render when cpsRecommendations toggle is enab
   }
 }
 
-.emotion-6:not(:focus):not(:active) {
-  -webkit-clip-path: inset(100%);
-  clip-path: inset(100%);
-  clip: rect(1px, 1px, 1px, 1px);
-  height: 1px;
-  overflow: hidden;
-  position: absolute;
-  width: 1px;
-  margin: 0;
-}
-
```

Comment:
This looks like the SkipLink CSS has been removed, which is not what we want...

Reply:
Snapshots updated now! This has been fixed and reintroduced.